IDEAS/BoundaryConditions/Interfaces/PartialSimInfoManager.mo

```diff
@@ -101,18 +104,23 @@ partial model PartialSimInfoManager
     annotation(Dialog(enable=interZonalAirFlowType<>
     IDEAS.BoundaryConditions.Types.InterZonalAirFlow.None or unify_n50,group="Interzonal airflow"));
 
+  parameter Boolean  use_sim_Cs =true "if checked, the default Cs of each surface in the building is sim.Cs"
+  annotation(choices(checkBox=true),Dialog(group="Wind"));
+
   parameter Modelica.Units.SI.Length H=10 "Building or roof height"
     annotation (Dialog(group="Wind"));
-  parameter Real A0=0.6 "Local terrain constant. 0.6 for Suburban,0.35 for Urban and 1 for Unshielded (Ashrae 1993) " annotation(Dialog(group="Wind"));
+  parameter Real A0=0.6 "Local terrain constant. 0.6 for Suburban,0.35 for Urban and 1 for Unshielded (Ashrae 1993) "
+    annotation(Dialog(group="Wind"));
   parameter Real a=0.28 "Velocity profile exponent. 0.28 for Suburban, 0.4 for Urban and 0.15 for Unshielded (Ashrae 1993) "
-                                                                                                                            annotation(Dialog(group="Wind"));
-  parameter Modelica.Units.SI.Length Hwin=10
+    annotation(Dialog(group="Wind"));
+  parameter Modelica.Units.SI.Length Hwind=10
```

Comment:
@Mathadon, this is something you've changed at soms point. Any perticular reason for this? 

Anyway, if 4 letters is clearer  then 3, why limit?

Reply:
@kldjonge the reason I indicated this is because of the [Buildings (and IBPSA) style guide](https://simulationresearch.lbl.gov/modelica/releases/v10.0.0/help/Buildings_UsersGuide.html#Buildings.UsersGuide.Conventions). The IDEAS library definitely does not follow that guideline strictly, but maybe we could take it into account for future developments and changes.

Nevertheless, if both of you would like to stick to 4 letters, we are of course free to keep it that way :)

Reply:
@jelgerjansen see https://github.com/open-ideas/IDEAS/pull/1398